### PR TITLE
Bug linux capitals

### PR DESCRIPTION
**Pull request by jaredbroad**
### What does this implement/fix? Explain your changes.
Address this error:
```/home/pi/pythonnet/src/console/Console.csproj(97,3): error MSB4019: The imported project "/usr/lib/mono/msbuild/Current/bin/Microsoft.CSHARP.Targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
/home/pi/pythonnet/src/embed_tests/Python.EmbeddingTest.csproj(118,3): error MSB4019: The imported project "/usr/lib/mono/msbuild/Current/bin/Microsoft.CSHARP.Targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
/home/pi/pythonnet/src/testing/Python.Test.csproj(104,3): error MSB4019: The imported project "/usr/lib/mono/msbuild/Current/bin/Microsoft.CSHARP.Targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
/home/pi/pythonnet/src/runtime/Python.Runtime.csproj(161,3): error MSB4019: The imported project "/usr/lib/mono/msbuild/Current/bin/Microsoft.CSHARP.Targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.```


### Does this close any currently open issues?
No

### Any other comments?
No

### CODE REVIEW

**Comment by jaredbroad** [comment]
For later mono versions they renamed the file to lower case

File: src/runtime/interop36m.cs [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+#if PYTHON36
+using System;
+using System.Collections;
+using System.Collections.Specialized;
+using System.Runtime.InteropServices;
+using System.Reflection;
+using System.Text;
+
+namespace Python.Runtime
```
Comment (Martin-Molinero) [inline src/runtime/interop36m.cs:13]: Think the `m` versions of the interop files are generated at runtime?

File: src/console/Console.csproj [hunk 2]
```diff
@@ -94,8 +94,8 @@
       <Name>Python.Runtime</Name>
     </ProjectReference>
   </ItemGroup>
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSHARP.Targets" />
+  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
```
Comment (Martin-Molinero) [inline src/console/Console.csproj:97]: 💯

File: src/runtime/interop36m.cs (hunk 1 above)
Comment (AlexCatarino) [reply src/runtime/interop36m.cs:13]: The interop files are generated during the installation process:
https://github.com/QuantConnect/pythonnet/blob/master/setup.py#L220

**Comment by jaredbroad** [comment]
Closing for you to merge in via 2.5

**Pull request closed without merging**